doc/doc-site/docs/overview/orders.md

```diff
@@ -250,7 +250,7 @@ One increment or decrement in order size should be a meaningful but not overwhel
 
 Let's create a market with these key properties in mind, and find a configuration that makes sense in light of these concerns.
 
-### Creating a hypothetical market
+### Creating hypothetical markets
```

Comment:
I can't comment on the section directly since it doesn't have changes in this PR, but the new content changes the context a little bit so I'm commenting here

In the part that starts, "Configuring a market requires three values beyond the base and quote type:", can you add in an explanation about how the market registration process (e.g. the Move code) operates in integers? Basically for the table with the 3 rows lot size, tick size, and minimum order size, it isn't immediately clear that these are integer config values that the Move package expects

Reply:
done